### ISSUE TITLE
Heading spacing after details / expander component

### ISSUE THREAD

**Issue by davidhunter08** [nhsuk/nhsuk-frontend#486]
# Bug Report

## What is the issue?

The spacing between a `details` component and a `h2` / `nhsuk-heading-l` isn't right. The `h2` / `nhsuk-heading-l` needs `padding-top` adding.

Similar to how it works for paragraphs:

```
%nhsuk-body-m  + %nhsuk-heading-l,
%nhsuk-body-s  + %nhsuk-heading-l,
%nhsuk-list + %nhsuk-heading-l {
  @include nhsuk-responsive-padding(4, 'top');
}
```

## What steps are required to reproduce the issue?

Add a `details` component to a page, followed by a `h2`.

You will see that there is no `padding-top` applied to the `h2`.

<img width="644" alt="Screenshot 2019-07-03 at 17 02 12" src="https://user-images.githubusercontent.com/14331000/60608710-03837180-9db8-11e9-8769-f62130f93699.png">

It should look like this:

<img width="644" alt="Screenshot 2019-07-03 at 17 02 27" src="https://user-images.githubusercontent.com/14331000/60608730-0d0cd980-9db8-11e9-944a-e0c3f52f2055.png">

**Comment by AdamChrimes** [comment]
The code for this fix is in the NHS digital service manual: https://github.com/nhsuk/nhsuk-service-manual/blob/develop/app/styles/main.scss#L60

**Comment by AdamChrimes** [comment]
Fixed in v2.3.0.

Thanks @Lily418 :-)
